src/components/Weather/Weather.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```

Comment:
Can we rename this component to`WeatherView.js` to clearly state that it is a presentation component